soh/soh/GameMenuBar.cpp

```diff
@@ -1139,6 +1139,11 @@ namespace GameMenuBar {
             }
             UIWidgets::Tooltip("Switches Link's age and reloads the area.");   
 
+            if (ImGui::Button("Clear Cutscene Pointer")) {
+                CVarSetInteger("gClearCutscenePointer", 1);
```

Comment:
Instead of having this set a CVar and having the function run every frame, couldn't the reset not be in a raw GI action and just be called directly from the button?

Reply:
yea that sounds much better, hadnt seen the raw action game-interactor stuff before